source/modules/MEI.stringtab.xml

```diff
@@ -1,51 +1,126 @@
+  <macroSpec ident="data.COURSENUMBER" module="MEI.stringtab" type="dt">
+    <desc xml:lang="en">In string tablature, the number of the course to be played, i.e., [1-9]+.</desc>
+    <content>
+      <rng:data type="positiveInteger" />
+    </content>
+  </macroSpec>
+  <macroSpec ident="data.COURSETUNING" module="MEI.stringtab" type="dt">
+    <desc xml:lang="en">Standard course tunings.</desc>
+    <content>
+      <valList type="semi">
+        <valItem ident="guitar.standard">
+          <desc xml:lang="en">Standard tuning for current guitars. The courses are tuned to E2 A2 D3
+            G3 B3 E4.</desc>
+        </valItem>
+        <valItem ident="guitar.drop.D">
+          <desc xml:lang="en">Drop D tuning for guitars. The lowest course is tuned down to D, while
+            all other courses are kept to their regular pitches. D2 A2 D3 G3 B3 E4.</desc>
+        </valItem>
+        <valItem ident="guitar.open.D">
+          <desc xml:lang="en">Open D tuning for guitars. D2 A2 D3 F3s A3 D4.</desc>
+        </valItem>
+        <valItem ident="guitar.open.G">
+          <desc xml:lang="en">Open G tuning for guitars. D2 G2 D2 G2 B3 D4.</desc>
+        </valItem>
+        <valItem ident="guitar.open.A">
+          <desc xml:lang="en">Open A tuning for guitars. E2 A2 E3 A3 C4s E4.</desc>
+        </valItem>
+        <valItem ident="lute.renaissance.6">
+          <desc xml:lang="en">Renaissance tuning for lutes with 6 courses. G2 C3
+            F3 A3 D4 G4.</desc>
+        </valItem>
+        <valItem ident="lute.baroque.d.minor">
+          <desc xml:lang="en">Baroque tuning for lutes with 6 stable courses, and additional bass
+            courses tuned to the key of D minor. A2 D3 F3 A3 D4 F4 for the main six courses, and bass courses descending in pitch from G2, F2, etc., depending on the size of the instrument. </desc>
+        </valItem>
+      </valList>
+    </content>
+  </macroSpec>
+  <classSpec ident="att.course.log" module="MEI.stringtab" type="atts">
+    <desc xml:lang="en">Logical domain attributes.</desc>
+    <classes>
+      <memberOf key="att.pitched"/>
+    </classes>
+  </classSpec>
+  <classSpec ident="att.scoreDef.vis.tablature" module="MEI.stringtab" type="atts">
+    <desc xml:lang="en">Visual domain attributes for scoreDef in the tablature repertoire.</desc>
+  </classSpec>
   <classSpec ident="att.stringtab" module="MEI.stringtab" type="atts">
     <desc xml:lang="en">String tablature string and fret information.</desc>
     <attList>
+      <attDef ident="tab.align" usage="opt">
+        <desc xml:lang="en">Attribute that describes the vertical alignment of tablature symbols.
+          Only applicable in cases where the symbols' vertical position does not communicate other
+          information, such as courses (i.e., only in German lute tablature). Typical values are <val>
+            top</val> and <val>bottom</val>.</desc>
+        <datatype>
+          <rng:ref name="data.VERTICALALIGNMENT" />
```

Comment:
```suggestion
          <rng:ref name="data.VERTICALALIGNMENT"/>
```

Reply:
No, we did not yet. However, the suggestions make it at least a little bit consistent with most of the schema. (In some places, it was actively changed with this PR for no obvious reason, so these suggestions just move things back.)

Don't see a reason why we should merge it with "wrong" formatting now, just to correct it in another PR?